lib/cli/generators/RIOT/template/stories/index.stories.js

```diff
@@ -0,0 +1,25 @@
+import { mount, storiesOf } from '@storybook/riot';
+import { linkTo } from '@storybook/addon-links';
+
+/* eslint-disable-next-line import/no-webpack-loader-syntax */
+import MyButtonRaw from 'raw-loader!./MyButton.tag';
+/* eslint-disable-next-line no-unused-vars */
+import MyButton from './MyButton.tag';
+/* eslint-disable-next-line no-unused-vars */
+import Welcome from './Welcome.tag';
+
+storiesOf('Welcome', module).add('to Storybook', () =>
+  mount('welcome', { showApp: () => linkTo('Button') })
```

Comment:
I noticed've that `mount` is used to add params to the component. Is it true? Sorry if I am asking the same questions 😅 
Can't we just return from story something like this:

```js
storiesOf('Welcome', module).add('to Storybook', () => {
  return {
    name: 'welcome',   // tag ?
    data:  { 
      showApp: () => linkTo('Button') 
    }
  }
}
```

Reply:
yes, exactly, we use mount to do that,
and that is a landmark for the riot developers when they see mount, 
they will feel like the framework does nothing else than a proxy to riot.

anyway this looks like another good idea : give the possibility to return only the params without mount (but users should be able to use mount anyway)

Reply:
in the riot jargon, it should be :

```javascript
storiesOf('Welcome', module).add('to Storybook', () =>({
  tagName: 'welcome',
  opts: {
    showApp: () => linkTo('Button') 
  }))
```

Reply:
done, added a jest test and a kitchen sink example